lazy_theta_star_planner/include/lazy_theta_star_planner/lazy_theta_star2.h

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 Anshumaan Singh
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Consider the functions in this header how many of them can be const references rather than just references, to protect people editing this later from doing something silly